src/test/java/io/rebloom/client/TDigestTest.java

```diff
@@ -162,17 +163,21 @@ public void minAndMax() {
 //    assertTrue((Object) cl.tdigestMin(key) instanceof Double);
 //    assertTrue((Object) cl.tdigestMax(key) instanceof Double);
 
-    cl.tdigestAdd(key, definedAddParam(2, 1));
-    cl.tdigestAdd(key, definedAddParam(5, 1));
+    cl.tdigestAdd(key, 2);
+    cl.tdigestAdd(key, 5);
     assertEquals(2d, cl.tdigestMin(key), 0.01);
     assertEquals(5d, cl.tdigestMax(key), 0.01);
   }
-
-  private static TDigestValueWeight randomAddParam() {
-    return new TDigestValueWeight(random.nextDouble() * 10000, Math.abs(random.nextInt()) + 1l);
-  }
-
-  private static TDigestValueWeight definedAddParam(double value, long weight) {
-    return new TDigestValueWeight(value, weight);
+//
```

Comment:
leftover?

Reply:
@filipecosta90 Yes, just in case an optional WEIGHT argument is added in near future :wink: